reframe/frontend/cli.py

```diff
@@ -81,6 +84,12 @@ def main():
         help='Load checks recursively')
 
     # Select options
+    select_options.add_argument(
+        '--modules-mapping', action='store', dest='module_map_file',
+        help='Map a module name to a different one')
+    select_options.add_argument(
+        '--try-module', action='append', dest='module_map_cl', default=[],
+        help='Map a module name to a different one')
```

Comment:
If we keep the `--try-module` name for the option, we should  perhaps adapt a bit this help message.